core/src/main/java/io/questdb/griffin/CompiledQuery.java

```diff
@@ -29,27 +29,27 @@
 import io.questdb.cutlass.text.TextLoader;
 
 public interface CompiledQuery {
-    int SELECT = 0;
```

Comment:
why change?

Reply:
Because ultimately the event that I store is a short. I looked at this and all we use is 1 to 14, I thought we might save the extra space and use a short too since it is very unlikely we will ever need bigger than that. I can revert though, let me know.

Reply:
It didn’t click with me at the time that these are the values you store. It is ok to change type!